services/workflows-service/src/workflow/workflow.service.unit.test.ts

```diff
@@ -224,171 +225,4 @@ describe('WorkflowService', () => {
       expect(definitions[0]).not.toHaveProperty('updatedAt');
     });
   });
-
-  describe('.event', () => {
```

Comment:
It was too difficult to fix these tests as it is a unit test and not an integration/e2e test. Now we're working with DB transactions so it makes everything harder to mock.